provisioning/standalone/docs/gettingstarted.md

```diff
@@ -0,0 +1,45 @@
+# Standalone Applications
+
+## How to create a standalone application?
+
+To create a container that can be used  as a **standalone application**, some simple rules have to be followed:
+
+-   The **port** to which the service is exposed has to be set using the environment variable `$CROWNLABS_LISTEN_PORT`.
+-   The **user** running the container cannot be **root**.
```

Comment:
Would say that it should have UID 1010 (IIRC), and that relevant files must be accessible by that user.